MapboxDirections.swift

```diff
@@ -21,106 +21,197 @@ public class MBPoint {
 
 }
 
+private func CLLocationCoordinate2DFromJSONArray(array: [Double]) -> CLLocationCoordinate2D? {
+    guard array.count == 2 else {
+        return nil
```

Comment:
It shouldn’t, so I made this an assertion instead. At least until the v5 endpoint stabilizes, it’s better for us to fail early and predictably than to fail gracefully in the face of a malformed response.